src/providers/incoming-data/incoming-data.ts

```diff
@@ -726,26 +722,6 @@ export class IncomingDataProvider {
     return newUri;
   }
 
-  public getCoinFromUri(data: string): Coin {
-    let coin = Coin.BTC;
-    const protocol = data.split(':')[0];
-    switch (protocol) {
-      case 'bitcoin':
-        coin = Coin.BTC;
-        break;
-      case 'bitcoincash':
-        coin = Coin.BCH;
-        break;
-      case 'ethereum':
-        coin = Coin.ETH;
-        break;
-      default:
-        coin = Coin.BTC;
-        break;
-    }
-    return coin;
-  }
-
```

Comment:
deprecated way to determine coin. This will force us to register `pax:`, `usdc:`, `gusd:` prefix per ERC20 token. We can use `ethereum:` for all erc20 tokens and fetch `getPaymentOptions` to determine coin,